test/integration/test-example-validation.js

```diff
@@ -54,6 +54,7 @@ describe('example', function() {
     'instagram.amp.html',
     'released.amp.html',
     'soundcloud.amp.html',
+    'springboard-player.amp.html',
```

Comment:
Just add a line here gets your test passing. https://github.com/ampproject/amphtml/blob/master/test/integration/test-example-validation.js#L72

Please also file an issue for validation to be implemented by us.